src/transformers/modeling_utils.py

```diff
@@ -1927,6 +1942,9 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             subfolder (`str`, *optional*, defaults to `""`):
                 In case the relevant files are located inside a subfolder of the model repo on huggingface.co, you can
                 specify the folder name here.
+            variant (`str`, *optional*):
+                If specified load weights from `variant` filename, *e.g.* pytorch_model.<variant>.bin. `variant` is
+                ignored when using `from_tf` or `from_flax`. available an error is thrown.
```

Comment:
I don't get what you wanted to say in the last sentence here.

Reply:
I suggested the "`variant` is ignored..." bit, but the trailing sentence indeed looks strange.

```suggestion
                ignored when using `from_tf` or `from_flax`.
```